README.md

```diff
@@ -107,6 +107,25 @@ $rows = SimpleExcelReader::create($pathToCsv)
 });
 ```
 
+#### Manually setting the headers
+
+If you would like to use a specific array of values for the headers, you can use the `setHeaders()` method.
+
+```php
+// $rows is an instance of Illuminate\Support\LazyCollection
+$rows = SimpleExcelReader::create($pathToCsv)
+    ->setHeaders(['email_address', 'given_name'])
```

Comment:
let's rename this to `useHeaders`, feels a tad more natural.

Reply:
Great, I have made this change.